elements/rh-footer/rh-footer.ts

```diff
@@ -1,18 +1,210 @@
-import './rh-footer-social-link.js';
-import './rh-footer-links.js';
-import './rh-footer-block.js';
-import './rh-footer-universal.js';
+import { LitElement, html } from 'lit';
+import { property } from 'lit/decorators/property.js';
+import { customElement } from 'lit/decorators/custom-element.js';
+import { classMap } from 'lit/directives/class-map.js';
 
+import { getRandomId } from '@patternfly/pfe-core/functions/random.js';
+import { Logger } from '@patternfly/pfe-core/controllers/logger.js';
 import '@patternfly/elements/pf-icon/pf-icon.js';
 import '@rhds/elements/rh-accordion/rh-accordion.js';
 
-import { RhFooter } from './RhFooter.js';
-import { RhFooterUniversal } from './rh-footer-universal.js';
-import { RhGlobalFooter } from './rh-global-footer.js';
+export { RhFooterUniversal } from './rh-footer-universal.js';
+export { RhGlobalFooter } from './rh-global-footer.js';
+export { RhFooterSocialLink } from './rh-footer-social-link.js';
+export { RhFooterLinks } from './rh-footer-links.js';
+export { RhFooterBlock } from './rh-footer-block.js';
```

Comment:
I'm not sure if we should do this or not.  I don't really see a downside to exporting all of these classes. 🤷‍♂️

Reply:
we'd have to support them at these paths. better not to export them and treat them as 'private'

Reply:
maybe keep `RhFooterUniversal` though